webextensions/manifest/commands.json

```diff
@@ -247,7 +247,7 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": true,
+                "version_added": "86",
```

Comment:
This should probably be 25, along with commands itself (which is currently ≤54). See https://web.archive.org/web/20130824074541/https://developer.chrome.com/extensions/commands.html